### PR TITLE
Remove hardcoded amd64 from Dockerfile for MultiArch builds

### PR DESCRIPTION
**Pull request by jeffdyoung**
Remove hardcoded amd64 from Dockerfile for MultiArch builds.

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -1,12 +1,14 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/console-operator
 COPY . .
-RUN ADDITIONAL_GOTAGS="ocp" make build WHAT="cmd/console"
+RUN ADDITIONAL_GOTAGS="ocp" make build WHAT="cmd/console"; \
```
Comment (benjaminapetersen) [inline Dockerfile.rhel7]: I'm fine with removing `amd64`, let's make sure we are consistent with other operators, however.

Comment (benjaminapetersen) [reply Dockerfile.rhel7]: (I imagine this step should be pretty standardized)

Comment (jeffdyoung) [reply Dockerfile.rhel7]: @benjaminapetersen  There seems to be 2 ways to approach this:

Build output to w/o arch in the output path:
https://github.com/openshift/cluster-autoscaler-operator/blob/master/Dockerfile.rhel7
https://github.com/openshift/cluster-samples-operator/blob/master/Dockerfile.rhel7

Build output w/arch in the output path:
https://github.com/openshift/origin/pull/22410
https://github.com/openshift/prom-label-proxy/pull/7
https://github.com/openshift/cluster-capacity/pull/20/files

^ for the ones that do use arch-specific paths, this is the fix we've been using.

Having gone through 30ish of these repos, there doesn't seems to be a right/wrong/preferred way.. just repo owner's preference.

Comment (benjaminapetersen) [reply Dockerfile.rhel7]: Great, works for me.

**Comment by benjaminapetersen** [comment]
/retest

`release "release-latest" failed: pod release-latest was already deleted`

**Pull request merged**